cpp/src/dataframe/table.cu

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
This file is superseded by https://github.com/rapidsai/cudf/blob/branch-0.7/cpp/src/table/table.cpp and should be removed.